source/solvers/navier_stokes_base.cc

```diff
@@ -558,6 +558,7 @@ NavierStokesBase<dim, VectorType, DofsType>::iterate()
                           simulation_parameters.simulation_control.method);
       multiphysics->percolate_time_vectors(false);
 
+      announce_string(this->pcout, "Fluid Dymamics");
```

Comment:
This is quite a sad typo, all tests will have to be rechanged :')
```suggestion
      announce_string(this->pcout, "Fluid Dynamics");
```

Reply:
No worries, that takes 5 seconds really